### PR TITLE
ref(unreal): Move Unreal to top-level platform 

### PR DESCRIPTION
**Pull request by GloryOfNight**

### CODE REVIEW

**Comment by bruno-garcia** [comment]
Bump platformicons once this is merged and shipped: https://github.com/getsentry/platformicons/pull/52

File: src/wizard/unreal/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Unreal Engine
+doc_link: https://docs.sentry.io/platforms/unreal/
+support_level: production
+type: framework
```
Comment (jan-auer) [inline src/wizard/unreal/index.md:5]: Please place redirects from the old root pages here, and make sure to add redirects for all the moved pages. Apart from links in Sentry docs, there can be blog posts, internal documentation, and bookmarks pointing to those URLs.

```yaml
redirect_from:
  - /platforms/unrealengine/
  - /platforms/native/ue4/
  - /platforms/native/guides/ue4/
```

**Comment by bruno-garcia** [comment]
The unreal logo should show up once you pull `master` into your branch

**Comment by GloryOfNight** [comment]
Re-created PR from right branch:
https://github.com/getsentry/sentry-docs/pull/4856

**Pull request closed without merging**